### ISSUE TITLE
Unable to use Tiny Spells insert replacement word functionality when using NVDA in certin applications i.e. notepad

### ISSUE THREAD

**Issue by Ednunp** [nvaccess/nvda#8483]
When using <!--any build of NVDA although I am currently using the latest 
 alpha 15 495, I am unable to use the insert replacement word functionality of Tiny Spell, whilst inside notepad.

To recreate:
1. Download the free Tiny spell and activate NVDA.
2. open the standard Notepad (I am using windows 7).
3. Type a misspelled word into notepad and hit space. Tiny spell should bring up a spelling tip and beep if set correctly.
4. Open the list of replacement words. This can be done by hitting the down arrow key straight after pressing the space bar if no mofifications have been made to the keyboard shortcuts.
5. Scroll through the words you want to replace with using the up and down arrow keys. Press enter to insert the new word.
6. the word should be inserted, but is not. In other applications, this works fine. With NVDA off, it works fine. The word is also coppied to the clipboard so this functions as a sort of workaround for now.

Thanks.

Ed.

**Comment by josephsl** [comment]
Hi, can you try it with Settings/Keyboard/Handle keys from other applications checkbox unchecked? Thanks (and by the way, I’d be happy to reformat the original comment to comply with the issue template).

**Comment by Ednunp** [comment]
HI. I have just tried that and it does fix the problem thank you. Though I am told this might break other things by turning it off. What is the correct solution here/ does a fix need to be implemented? also apologies about the formatting with the issue template, I've not raised issues on here for a very long time.

**Comment by josephsl** [comment]
Hi, the ideal solution is a configuration profile where this setting is off while using this overlay. Depending on how this overlay is shown (an app or not an app at all), it’ll change what kind of profile you can create. Thanks.

**Comment by ehollig** [comment]
Closing as worksforme from https://github.com/nvaccess/nvda/issues/8483#issuecomment-403027710
